pheget/api/__init__.py

```diff
@@ -2,13 +2,73 @@
 API endpoints (return JSON, not HTML)
 """
 
+import json
+import sqlite3
+import urllib.request
+
 from flask import Blueprint, abort, jsonify, request
 
+from .. import model
 from .format import query_variants
 
 api_blueprint = Blueprint("api", __name__)
 
 
+@api_blueprint.route("/bestvar/<string:symbol>/", methods=["GET"])
+def gene_query(symbol: str):
+    """
+    Given a gene, query an SQL database to find the variant with the strongest association with that gene
+    """
+    # ~UGLY HACK~ If omnisearch did not return a gene_id, then searchbox will fill in "undefined" - immediately abort in that case
+    if symbol == "undefined":
```

Comment:
If this still necessary, in light of this code?

https://github.com/statgen/pheget/pull/47/files#diff-fb9be9e8a12a6fe4420134c94525c670R73

The basic principle is to avoid making requests you think are unnecessary. (performance wonks like to say "the fastest request is the one not made")

If JS gives a value of `undefined`, we wouldn't ever expect that value to be sent to the server at all.

Reply:
I removed this, it should work fine without.